src/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,115 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Plots {
+  export class ErrorBar<X, Y> extends XYPlot<X, Y> {
+
+    private _tickLength = 10;
+
+    constructor() {
+      super();
+      this.addClass("error-bar-plot");
+    }
+
+    /**
+     * Sets the tick length
+     */
+    public tickLength(): number;
+    public tickLength(tickLength: number): ErrorBar<X, Y>;
+    public tickLength(tickLength?: number): any {
```

Comment:
What if I want to set the top and bottom tick lengths independently? Is that something we should support?